boards/v2/google/google_kukui/google_kukui_defconfig

```diff
@@ -1,12 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Zephyr Kernel Configuration
```

Comment:
Empty heading can go

Reply:
Those commits are direct outputs from the conversion script. 
So I've seen this, but I was planning to clean them up in a single commit once all boards are converted.
I though this could be acceptable as long as those are only comments and not the Kconfig functional part